fmriprep/workflows/bold/outputs.py

```diff
@@ -256,60 +256,133 @@ def init_func_derivatives_wf(
 
     fs_outputs = spaces.cached.get_fs_spaces()
     if freesurfer and fs_outputs:
-        from niworkflows.interfaces.surf import GiftiNameSource
```

Comment:
I haven't removed this for backwards compatibility, but the new Path2BIDS should simplify its logic (no need anymore for building up the new pattern).